README.md

```diff
@@ -132,3 +132,49 @@ For more information on the Akamai API, read the [property manager docs](https:/
 ## Build Process
 
 This repository has a webhook that automatically builds a [Jenkins job](https://jenkins-jenkins.5a9f.insights-dev.openshiftapps.com/job/akamai-config-deployer/) on every push. To configure this webhook, check the project's [webhook settings](https://github.com/RedHatInsights/cloud-services-config/settings/hooks)
+
+## Testing your changes locally
+
+Testing local changes is straightforward. First, add a line like this to your insights-proxy spandx config:
+
+```diff
+--- a/profiles/local-frontend.js
++++ b/profiles/local-frontend.js
+@@ -9,5 +9,6 @@ routes[`/beta/${SECTION}/${APP_ID}`] = { host: `http://localhost:${FRONTEND_PORT
+ routes[`/${SECTION}/${APP_ID}`]      = { host: `http://localhost:${FRONTEND_PORT}` };
+ routes[`/beta/apps/${APP_ID}`]       = { host: `http://localhost:${FRONTEND_PORT}` };
+ routes[`/apps/${APP_ID}`]            = { host: `http://localhost:${FRONTEND_PORT}` };
++routes[`/beta/config`]            = { host: `http://localhost:8889` };
+
+ module.exports = { routes };
+```
+
+Restart your insights-proxy to pick up the change.
+
+Create a `beta/config` directory inside of `cloud-services-config` and copy `main.yml` to it. Then, from the directory that the `beta` dir lives in, run `python -m SimpleHTTPServer 8889`. In your browser, go to `https://ci.foo.redhat.com:1337/beta/rhel/dashboard`. You should see something logged like this from SimpleHTTPServer:
```

Comment:
> from the directory that the `beta` dir lives in

The wording here threw me for a bit as I was running the `SimpleHTTPServer ` from within `beta/config`. Suggestion: "from the `cloud-services-config` root directory run `python -m ...`"